apps/console/src/index.jsp

```diff
@@ -16,36 +16,101 @@
-<%= htmlWebpackPlugin.options.importSuperTenantConstant %>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1"%>
+<%@ page import="static org.wso2.carbon.identity.core.util.IdentityUtil.getServerURL" %>
 
-<jsp:scriptlet>
-    session.setAttribute("authCode",request.getParameter("code"));
-    session.setAttribute("sessionState", request.getParameter("session_state"));
-</jsp:scriptlet>
+<%
+    String serverUrl = getServerURL("", true, true);
+    String authorizationCode = request.getParameter("code");
+    String authSessionState = request.getParameter("session_state");
+    String authIdPs = request.getParameter("AuthenticatedIdPs");
+%>
 
 <!DOCTYPE HTML>
 <html>
     <head>
-        <%= htmlWebpackPlugin.options.contentType %>
-        <meta charset="utf-8"/>
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
-        <meta name="referrer" content="no-referrer" />
-
-        <link href="<%= htmlWebpackPlugin.options.publicPath %>/libs/themes/default/theme.<%= htmlWebpackPlugin.options.themeHash %>.min.css" rel="stylesheet" type="text/css"/>
-        <link rel="shortcut icon" href="<%= htmlWebpackPlugin.options.publicPath %>/libs/themes/default/assets/images/branding/favicon.ico" />
-        <script>
-            var contextPathGlobal = "<%= htmlWebpackPlugin.options.publicPath %>";
-            var serverOriginGlobal = "<%= htmlWebpackPlugin.options.serverUrl %>";
-            var superTenantGlobal = "<%= htmlWebpackPlugin.options.superTenantConstant %>";
-            var tenantPrefixGlobal = "<%= htmlWebpackPlugin.options.tenantPrefix %>";
-        </script>
+        <script src="https://unpkg.com/@asgardeo/auth-spa@0.2.19/dist/asgardeo-spa.production.min.js"></script>
     </head>
     <body>
-        <noscript>
-            You need to enable JavaScript to run this app.
-        </noscript>
-        <div id="root"></div>
+        <script>
+            var userAccessedPath = window.location.href;
+            var applicationDomain = window.location.origin;
+
+            var userTenant = userAccessedPath.split("/t/")[1] ?  userAccessedPath.split("/t/")[1].split("/")[0] : null;
+            userTenant = userTenant ?  userTenant.split("?")[0] : null;
+
+            var serverOrigin = "<%=serverUrl%>";
+            var authorizationCode = "<%=authorizationCode%>" != "null" ? "<%=authorizationCode%>" : null;
+            var authSessionState = "<%=authSessionState%>" != "null" ? "<%=authSessionState%>" : null;
+            var authIdPs = "<%=authIdPs%>" != "null" ? "<%=authIdPs%>" : null;
+            
+            function getApiPath(path) {
+                if(path) {
+                    return serverOrigin + path;
+                }
+
+                return serverOrigin;
+            }
+            
+            var auth = AsgardeoAuth.AsgardeoSPAClient.getInstance();
+
+            var authConfig = {
```

Comment:
Let's see if we can get these values from deployment.config via webpack. This can be done in a followup PR.